### ISSUE TITLE
Hard crash on `Err.Raise` from an ActiveX DLL

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#560]
**Describe the bug**
If the DLL has an `Err.Raise` with the intention of returning an error to the calling code (VBA in this case), the project crashes hard. I don't think that's ideal. ;-) 

**To Reproduce**
1. Create an ActiveX DLL project
2. Add the following code (in my case, I used a COM-creatable class):
```
    Public Sub Init()
        Err.Raise 5, , "Oops."
    End Sub
```
3. In a VBA host (Access in my case), add reference
4. Set up a call to the `Init` (in my case, the form's `Load` event was used)
5. Go out in a blaze of glory. 💥 
 
**Expected behavior**
A standard VBA runtime error should be returned, rather than a crash.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.13.30

**Comment by WaynePhillipsEA** [comment]
Looks to be a 64-bit only bug.   Thanks for reporting.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.13.31.   Thanks!
